server/src/typeDefs/user.ts

```diff
@@ -3,11 +3,10 @@ import { gql } from 'apollo-server'
 export const user = gql`
   scalar DateTime
   type User {
-    email: String!
     id: ID!
-    name: String
-    role: Role
-    lastLogin: DateTime
+    fullName: String
+    email: String!
+    createdAt: DateTime!
```

Comment:
so here you could add `settings: UserSettings` as a subfield, and then retrieve it with a query like:
```
user {
   settings {
     theme
    }
}
```